requirements.txt

```diff
@@ -1,5 +1,5 @@
 pyyaml>=3.0,<=3.99
 click>=6.0,<=6.99
 jsonref>=0.1.0,<=0.1.99
-requests>=2.12.0,==2.*
+requests>=2.12.0,<2.14
```

Comment:
It doesn't look like our code has any incompatibility. It seems to be in the Travis environment. You can look at the errors on the first two commits -- something went wrong on the `pip install` step. I'm still trying to figure it out.